### DISCUSSION TITLE
Welcome to Commander_Wars Discussions!

### DISCUSSION THREAD

**Discussion by Robosturm** [Robosturm/Commander_Wars#690] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cinnamon189** [comment]
Hi folks,

first of all, thanks a lot to all contributors of this project! :-) The Advance Wars series is one of my favourites among turn-based strategy games, and I'm glad that now I can replay these classics, with some improvements. For me, AW2 hits the sweet spot in terms of complexity, though occasionally I also play with contents from DS, DC and CW.

I've playtested other games before and would like to occasionally contribute with bug reports and suggestions. Discord I only use like several times a year (since I have read their definition of privacy), so you will mostly find me on Github and, for voice chat, Mumble. That said, I'm a rather casual gamer, so I'm wary of online gaming... But maybe something will turn up after all. Until then, I'll just follow the discussions that arise here.

See you! :-)

**Reply by Robosturm** [reply]
Hi,
just a heads up no one really uses the discussions tap nor visists it.
So don't excpect any responses here.

**Reply by cinnamon189** [reply]
Thought so, but thank you for telling me :-) I'll still keep an eye on it.
